config/load.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"io/ioutil"
+
+	"github.com/jessevdk/go-flags"
+	"github.com/kelseyhightower/envconfig"
+)
+
+type ConfigPath struct {
+	FromYAML   bool   `long:"fromYaml" env:"FROM_YAML" description:"load configuration from yaml file otherwise from ENV"`
+	ConfigPath string `long:"config" env:"CONFIG" default:"config.yaml" description:"yaml config file path"`
```

Comment:
If it's path, change default to `/etc/plgd-config/cloud.yaml` so it's clear that it's not only

Reply:
default must be removed otherwise ConfigPath is not empty